### PR TITLE
add run id to build summary

### PR DESCRIPTION
**Pull request by dhuangnm**
## Summary:

Add run id to build summary

## Test Plan:

Workflow runs from other repos.

### CODE REVIEW

**Review by derekk-nm — DISMISSED** [review]
with this required, you'll need to update any uses in other repos using it.

**Comment by dhuangnm** [comment]
> with this required, you'll need to update any uses in other repos using it.

No we use tags so only the place where it needs this updated action needs to use it.

**Comment by dbarbuzzi** [comment]
> No we use tags so only the place where it needs this updated action needs to use it.

Yes, but if we ever need to update the step for other functionality fixes that those repos need, then they'd need to be updated that point.

Therefore, instead of taking `run_id` as an input, perhaps it should just use `github.run_id` and that can be a known limitation of the action? Alternatively, maybe this action gets the 'extra' field treatment instead of explicit `run_id` support, and thus the calling step can pass the `run_id` as extra info? Or just make `run_id` optional, and the script can omit this new line if it’s not defined?

File: actions/summary-build/action.yml [hunk 1]
```diff
@@ -12,18 +12,15 @@ inputs:
...
   steps:
     - run: |
-        if [ ${{ inputs.whl_status }} -eq 0 ]; then
-            # green check
-            WHL_EMOJI=$(echo -e "\xE2\x9C\x85")
-        else
-            # red x
-            WHL_EMOJI=$(echo -e "\xE2\x9D\x8C")
-        fi
+        WHL_EMOJI=`${{ github.action_path }}/../../scripts/step-status.sh ${{ inputs.whl_status }}`
```
Comment (andy-neuma) [inline actions/summary-build/action.yml:23]: i see the encapsulation here, which i think is okay. the bulk of the action predates "nm-actions" and our use of it.

File: actions/summary-build/action.yml [hunk 2]
```diff
@@ -12,18 +12,15 @@ inputs:
   whl_status:
     description: status from build step
     required: true
+  run_id:
```
Comment (andy-neuma) [inline actions/summary-build/action.yml:15]: stop passing in `run_id` it is not needed

**Review by andy-neuma — COMMENTED** [review]
thanks

**Pull request closed without merging**